mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -333,13 +384,42 @@ class KMeans @Since("1.5.0") (
   override def transformSchema(schema: StructType): StructType = {
     validateAndTransformSchema(schema)
   }
+
+  @Since("2.1.0")
+  override def write: MLWriter = new KMeans.KMeansWriter(this)
 }
 
 @Since("1.6.0")
 object KMeans extends DefaultParamsReadable[KMeans] {
 
   @Since("1.6.0")
   override def load(path: String): KMeans = super.load(path)
+
+  @Since("2.1.0")
+  override def read: MLReader[KMeans] = new KMeansReader
```

Comment:
This has the same signature as before, right?  If so, then the Since version can be 1.6.0.